### DISCUSSION TITLE
Learning OpenTelemetry - http://localhost:8080/feature - 404 Page not found

### DISCUSSION THREAD

**Discussion by punpradip** [open-telemetry/opentelemetry-demo#1521] [Q&A] [answered]
Hi,
I am going through chapter 4 of Oreily's Learning OpenTelemetry book. All the docker containers are running using Docker desktop and following the instructions. However, I am not able to access http://localhost:8080/feature. I get 404 Page not found. Is anyone having this issue? Please advise. Thanks.

**Comment by julianocosta89** [comment, marked as answer]
Hello @punpradip this endpoint was removed on the last release.
We have the new doc available here: https://opentelemetry.io/docs/demo/feature-flags/

**Reply by punpradip** [reply]
Thank you @julianocosta89
